src/crypto/public_key/rsa.js

```diff
@@ -140,8 +140,12 @@ export async function encrypt(data, n, e) {
  * @async
  */
 export async function decrypt(data, n, e, d, p, q, u, randomPayload) {
-  if (util.getNodeCrypto()) {
-    return nodeDecrypt(data, n, e, d, p, q, u, randomPayload);
+  if (util.getNodeCrypto() && !randomPayload) {
```

Comment:
This change seems redundant with the change in `nodeDecrypt` below.. is this one intentional?